t/lib/Bakesale.pm

```diff
@@ -53,6 +53,8 @@ package Bakesale::Test {
 
     $user1 = $user_rs->single({ id => $user1->id });
 
+    $user1->ix_create_base_state;
```

Comment:
Like, creating any row for a class where ix_is_account_base is true automatically calls this? We can but I *think* I tried that and there's not a clean way to do it. When DBIC calls new it passes in a hashref of attrs that we can pull a result source out of, but it feels like we're reaching into DBIC internals.

Up to you